packages/devtools_app/lib/src/app.dart

```diff
@@ -60,16 +61,32 @@ const debugEnableSampleScreen = false;
 // Disabled until VM developer mode functionality is added.
 const showVmDeveloperMode = false;
 
-/// Whether this DevTools build is external.
-bool isExternalBuild = true;
+/// If true, features under construction will be enabled.
+///
+/// By default, the features are off in release mode.
+/// To enable them pass the compilation flag
+/// `--dart-define=enable_experiments=true`.
+bool _kEnableExperiments = const bool.fromEnvironment('enable_experiments');
```

Comment:
I suggest we move this and the enableNonProdFeatures code back to feature flags. It logically makes sense to be in that file since that is where we define feature visibility.

Make the `FeatureFlags.enableNonProdFeatures` method do what the `_enableNonProdFeatures` method does here, which is check for release mode and experiment status.